modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -7,44 +7,52 @@ endif::[]
+// Nahuel--anything to add here?
+
+[[specifying_ip_address]]
+== Specifying IP Addresses to Blacklist in the Policy
+
+You can specify the IP addresses to block in the policy using the following ways:
+
+* Blacklist access based on the IP address of the request
+
+* Blacklist access based on the origin IP address of the x-Forwarded-For header
+
+=== Blacklist Access Based on the IP Address of the Request
+
+To blacklist access based on the IP address of the request, you can:
+
+* Define a specific IP address by enumerating it in the white space, for example, `192.168.1.1`.
+* Define a subset of addresses by identifying a subnet mask. 
 +
-. In Apply New Policy.
-. In Select Policy, choose IP Blacklist, and click Configure Policy.
-. Specify a list of IP addresses in the blacklist section of Apply IP Blacklist Policy.
+For example, `192.168.3.1/30` includes the consolidated range: `192.168.3.0 - 192.168.3.3`.
+* Define a whole range of IP addresses by specifying the relevant octets of the IP address that you want to permit. 
 +
-Specified addresses are blacklisted to access and interact with your API.
+For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
 
-You can specify an address or a range or addresses in the following ways:
+=== Blacklist Access Based on the Origin IP Address of the x-Forwarded-For Header
 
-* Define a specific IP address by enumerating it in the white space. For example, 192.168.1.1.
-* Define a subset of addresses by identifying a subnet mask. For example, 192.168.3.1/30 will include the consolidated range: 192.168.3.0 - 192.168.3.3.
-* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
+If the client connects to your API through an HTTP proxy or a load-balancer, you can blacklist the client's specific IP (the IP originating the request) instead of the address that will appear in the request.
 
+For example, suppose you want to blacklist `192.168.2.3` and the address of a client connecting through an HTTP proxy is `92.40.1.255`, the client requests appear with this public address.
 
-== Option 2: Blacklist access based on the origin IP address of the x-Forwarded-For header
-If the client connects to your API through an HTTP proxy or a load-balancer, you can blacklist the client's specific IP (the IP originating the request) instead of the address that will appear in the request.
-For example:
-Suppose you want to blacklist 192.168.2.3, the address of a client connecting through an HTTP proxy whose address is 92.40.1.255. The client requests appear with this public address.
-Typically, applications use the 'X-Forwarded-For' header to identify the origin IP addresses of a request that was redirected to your endpoint.
+//Nahuel--which public address. The previous para is not clear as to what we want to say. 
```

Comment:
So you try to access a protected resource. You have an IP assigned by your company's router, for example. Let's say it's `192.168.2.3` (-> private IP). But the company has 1 Internet provider and only one IP that can be used to surf the Internet. For example `92.40.1.255` (-> public IP). So you may always have a static IP `192.168.2.3` but the public IP address may change without notice, and you'd like to blacklist the original private IP. In this scenario, all outgoing communications from the company will appear as coming from `92.40.1.255`, no mattering the actual private IP where the message was originated.

Reply:
Done! Thanks!